loguru.cpp

```diff
@@ -794,6 +831,22 @@ namespace loguru
 		return true;
 	}
 
+	bool add_syslog(const char* app_name, Verbosity verbosity, int facility /* Default: LOG_USER */)
+	{
+#ifdef LOGURU_SYSLOG
+		if (app_name == nullptr) {
+			app_name = argv0_filename();
+		}
+		openlog(app_name, 0, facility);
+		add_callback("'syslog'", syslog_log, nullptr, verbosity, syslog_close, syslog_flush);
+
+		VLOG_F(g_internal_verbosity, "Logging to 'syslog' , verbosity: " LOGURU_FMT(d) "", verbosity);
+		return true;
+#else
+		VLOG_F(g_internal_verbosity, "syslog not implemented on this system. Request to install syslog logging ignored.");
+        return false;
```

Comment:
nit: weird indentation.

Also: this will probably generate warnings about unused arguments on windows. Would be good to silence those:

```
(void)app_name;
(void)verbosity;
(void)facility;

Reply:
Fixed